src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -71,7 +71,7 @@ export default class Aircraft {
         /* eslint-disable no-multi-spaces*/
         this._navigationLibrary = navigationLibrary;
         this.eid          = global.prop.aircraft.list.length;  // entity ID
-        this.position     = [0, 0];     // Aircraft Position, in km, relative to airport position
+        this.position     = null;       // Aircraft Position
```

Comment:
I concur. Changed those that use a `StaticPositionModel` to have `._positionModel`, with a `.positionModel()` getter to privatize and protect it from change, and left those that use a `PositionModel` with a public `.positionModel` property, to allow it to be modified.